File: src/main/java/upbrella/be/store/repository/BusinessHourRepository.java
package upbrella.be.store.repository;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;
import upbrella.be.store.entity.BusinessHour;

public interface BusinessHourRepository extends JpaRepository<BusinessHour, Long> {

    List<BusinessHour> findByStoreMetaId(Long storeMetaId);

    void deleteAllByStoreMetaId(Long storeMetaId);
}

File: src/main/kotlin/upbrella/be/store/repository/BusinessHourRepository.kt
package upbrella.be.store.repository

import org.springframework.data.jpa.repository.JpaRepository
import upbrella.be.store.entity.BusinessHour

interface BusinessHourRepository : JpaRepository<BusinessHour, Long> {
    fun findByStoreMetaId(storeMetaId: Long): List<BusinessHour>

    fun deleteAllByStoreMetaId(storeMetaId: Long)
}